src/components/pages/Profile.tsx

```diff
@@ -34,47 +46,27 @@ function ProfileHeader({ address }: { address: string }): JSX.Element {
   );
 }
 
-const generateRandomData = (genome: string): HoundInfo => {
```

Comment:
Are we no longer showing the randomized hound list on a user's profile? Did we actually manage to fill that with legitimate hounds now?

Reply:
Yup. Although for the time-being there isn't too much variation in the genome being used to create the hounds.

Reply:
Ah, that's great!